src/scripts/main.js

```diff
@@ -2,7 +2,18 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const result = document.querySelector('table');
 
-// write your code here
+for (const person of people) {
```

Comment:
use decomposition

Reply:
@Volodymyr-Melnyk-Fron10der What mean decomposition. I did not understood clearly.